services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -190,7 +190,7 @@ public void configure(Binder binder)
             binder.bind(JettyServerInitializer.class).to(QueryJettyServerInitializer.class);
             Jerseys.addResource(binder, QueryResource.class);
             LifecycleModule.register(binder, QueryResource.class);
-            LifecycleModule.register(binder, LookupReferencesManager.class);
+            binder.install(new LookupModule());
```

Comment:
well i can see it is done the same way for [caching](https://github.com/b-slim/druid/blob/4915f4cece6abdcec79b4957209030e4c5f14870/services/src/main/java/io/druid/cli/CliPeon.java#L171-L171) but will change it.